src/Quadmath.jl

```diff
@@ -204,6 +205,9 @@ function frexp(x::Float128)
     return y, Int(r[])
 end
 
+significand(x::Float128) = frexp(x)[1] * 2
+exponent(x::Float128) = frexp(x)[2] - 1
```

Comment:
Note that `significand` and `exponent` have slightly different error handling for zeros, Infs and NaNs

Reply:
yes -- I'll adjust the defs

Reply:
`significand` works with `0, Inf, NaN` as given (fixed `exponent`)